modules/ztp-intro-factory-install.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: CONCEPT
+[id="introduction-factory-install_pipeline_{context}"]
+= An introduction to the factory install pipeline
```

Comment:
Not critical, but I don't think I've seen other headings in our doc set that start with "An introduction to". I think I moreso see things like "<item> overview" or "About <item>". Would one of those work here?